terminus-ui/scss/helpers/_cursors.spec.scss

```diff
@@ -1,2 +1,79 @@
 @import 'true';
 @import './cursors';
+
+@include describe ('cursors') {
```

Comment:
Seems like we test all cursor types here together, then again individually below. Do we need both sets of tests?

Reply:
Right. This is testing the mixin, and the one below tests the function.